transport/rif_comms/node.py

```diff
@@ -240,7 +240,14 @@ def send_message(self, payload: str, recipient: Address):
         """
         Send text message through the RIF Comms client.
         """
-        self._comms_client.subscribe_to(recipient)
+        self.log.info(
+            "RIF Comms sending message", message_payload=payload.replace("\n", "\\n"), recipient=pex(recipient)
+        )
+        if not self._comms_client.is_subscribed_to(recipient):
+            self.log.info(
+                "RIF Comms sending message not subscribed to recipient. Subscribing...", recipient=pex(recipient)
```

Comment:
logs messages should not be capitalized nor use punctuation.

imo, they should also mention as little as possible about the rif comms node logic (such as: "i need to be subscribed from the comms node to send this message")
```suggestion
                "subscribing to recipient", transport="rif_comms", recipient=pex(recipient)
```